src/seedu/addressbook/ui/TextUi.java

```diff
@@ -91,6 +91,22 @@ public String getUserCommand() {
         return fullInputLine;
     }
 
+    public void loginInfo() {
+        boolean status = true;
+        while (status) {
+            System.out.println("Enter your Username: ");
+            String username = in.nextLine();
+            System.out.println("Enter your Password: ");
+            String password = in.nextLine();
+            if (username.equals("nicholas") && password.equals("nicholas")) {
```

Comment:
This can probably go into a separate method called verifyLogin() or something.

Reply:
Yes, the code in this method needs refactoring to satisfy SRP and SLAP. Formatting, Logic functionalities should not ideally be at the same method.